tailwind.config.js

```diff
@@ -11,20 +11,26 @@ export default {
       transparent: 'transparent',
       current: 'currentColor',
       white: '#ffffff',
+      cream: '#FEFAF3',
+      flan: '#f4ecde',
       black: '#000000',
-      disabled: '#BCBAC0',
-      'bg-black': '#232326',
-      'bg-black-active': '#363639',
-      blue: '#459ABA',
-      'blue-light': '#69B9CD',
+      disabled: '#C8CDD0',
+      'gray-3': '#C8CDD0',
+      'gray-2': '#828282',
+      'gray-1': '#4f4f4f',
+      'bright-blue': '#5E69FF',
+      'dusty-rose': '#A06B87',
+      info: '#2F80ED',
       green: '#6EE9D1',
-      'green-success': '#7CFA4D',
+      success: '#27AE60',
+      warning: '#F99A2A',
+      error: '#EB5757',
+      teal: '#55B1AB',
       divider: '#504D4D',
       overlay: 'rgba(57, 52, 52, 0.75)',
-      error: '#E70F0F',
     },
     width: {
-      modal: '420px',
+      modal: '480px',
       'single-block': '550px',
       dashboard: '1010px',
       full: '100%',
```

Comment:
It won't let me suggest that far down, but change

'blue-border': '8px 0 0 #69B9CD',
to 
'blue-border': '8px 0 0 #5E69FF',